src/main/java/frc/robot/subsystems/Manager.java

```diff
@@ -0,0 +1,202 @@
+package frc.robot.subsystems;
```

Comment:
Refactoring note: This doesn't need to be done for this PR, but sticking `reset()` in every single if statement is difficult to maintain and confusing. Manager should probably have a private `setState()` function that sets the state and resets relevant timers.